src/main/java/com/solvd/utils/FileReader.java

```diff
@@ -1,29 +1,39 @@
 package com.solvd.utils;
 
-import java.io.FileInputStream;
 import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Paths;
+import java.io.InputStream;
 import java.util.Properties;
 
 public class FileReader {
```

Comment:
FileReader as a class name is a bit misleading because:
it loads config properties and queries
A better name would be something like:
ConfigReader